docs/src/apimanual.md

```diff
@@ -845,6 +842,39 @@ twice the value of the `coefficients` field in the `VectorAffineFunction` for
 the corresponding rows. See [`PositiveSemidefiniteConeTriangle`](@ref) for
 details.
 
+#### Dual for convex problem with quadratic functions
+
+Given a program with quadratic functions:
+```math
+\begin{align*}
+& \min_{x \in \mathbb{R}^n} & \frac{1}{2}x^TQ_0x + a_0^T x + b_0
+\\
+& \;\;\text{s.t.} & \frac{1}{2}x^TQ_ix + a_i^T x + b_i & \in \mathcal{C}_i & i = 1 \ldots m
```

Comment:
Note that this notation only covers scalar cones (which is perfectly fine).

Reply:
Yes, but all the text also works for non-scalar ones, vector quadratic function is tricky to write in LaTeX so I thought the reader would guess that it generalize for any C_i even vector ones.